pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -161,26 +161,13 @@ func (cgc *containerGC) removeOldestNSandboxes(sandboxes []sandboxGCInfo, toRemo
 	// Remove from oldest to newest (last to first).
 	for i := len(sandboxes) - 1; i >= numToKeep; i-- {
 		if !sandboxes[i].active {
-			cgc.removeSandbox(sandboxes[i].id)
+			if err := cgc.manager.DeleteSandbox(sandboxes[i].id); err != nil {
+				klog.Errorf("Failed to remove sandbox %q: %v", sandboxes[i].id, err)
```

Comment:
see https://github.com/golang/go/wiki/CodeReviewComments#error-strings

```suggestion
				klog.Errorf("failed to remove sandbox %q: %v", sandboxes[i].id, err)
```

Reply:
since we are logging the error here, it is starting with capital letter.